### ISSUE TITLE
pump.io validator.sanitize issue (when installing from git-master)

### ISSUE THREAD

**Issue by larjona** [e14n/pump.io#1143]
As reported in https://io.mndet.net/mnd/note/CdQIljYISImjmgfIlx-QdQ :

> Something went wrong...
> 
> ```
TypeError: sanitize is not a function
 at /opt/pump.io/lib/app.js:344:37
at Function._.each._.forEach (/opt/pump.io/node_modules/underscore/underscore.js:158:9)
at defang (/opt/pump.io/lib/app.js:342:19)
at /opt/pump.io/lib/app.js:348:37
at Function._.each._.forEach (/opt/pump.io/node_modules/underscore/underscore.js:158:9)
at Object.defang (/opt/pump.io/lib/app.js:342:19)
at Object.eval (eval at <anonymous> (/opt/pump.io/node_modules/underscore/underscore.js:1454:20), <anonymous>:100:24)
at Object.template (/opt/pump.io/node_modules/underscore/underscore.js:1461:21)
at ServerResponse.res._render (/opt/pump.io/node_modules/express/lib/view.js:425:21)
at ServerResponse.res.render (/opt/pump.io/node_modules/express/lib/view.js:318:17)
 

More comments:

> Ok, I fixed it by replacing sanitize(value).escape() with validator.escape(value) but... Well, it looks like
> 
> "dependencies": { ...,  "validator": "4.4.x", ... }
> 
> in package.json file previously points to other sources.
> 
> Pump.io taken from git (master branch last commit), dependencies installed locally with cd pump.io && npm install. Node.js version 4.2.6, npm version 2.14.12

**Comment by mnd** [comment]
There git diff for pump.io repo

````
$ git diff origin/master
diff --git a/lib/app.js b/lib/app.js
index 3da3182..ccde259 100644
--- a/lib/app.js
+++ b/lib/app.js
@@ -53,7 +53,6 @@ var urlparse = require("url").parse,
     Image = require("./model/image").Image,
     Proxy = require("./model/proxy").Proxy,
     ActivitySpam = require("./activityspam"),
-    sanitize = validator.sanitize,
     Databank = databank.Databank,
     DatabankObject = databank.DatabankObject,
     DatabankStore = require("connect-databank")(express);
@@ -341,7 +340,7 @@ var makeApp = function(configBase, callback) {
                 var dup = _.clone(obj);
                 _.each(dup, function(value, name) {
                     if (name == "displayName" && _.isString(value)) {
-                        dup[name] = sanitize(value).escape();
+                        dup[name] = validator.escape(value);
                     } else if (_.isFunction(value)) {
                         delete dup[name];
                     } else if (_.isObject(value)) {
````

**Comment by profOnno** [comment]
Nice work,
I will add it to a test case.

**Comment by strugee** [comment]
Fixed in #1144
